src/containers/Form/FormLayout.tsx

```diff
@@ -578,6 +585,12 @@ export const FormLayout = ({
                 variant="outlined"
                 color="primary"
                 onClick={() => {
+                  if (additionalAction?.state === 'template') {
```

Comment:
If possible can we make a generic way to update this and not hardcode template inside formlayout component